docs/pdfs.md

```diff
@@ -5,8 +5,14 @@ sidebar_label: Editing PDFs
 slug: /pdfs
 ---
 
-# Tips for working with PDF files
+## Add labels and fields
+We recommend using [Documate](https://documate.wrkflows.io/). It can do everything that AssemblyLine makes use of as far as PDF labels and fields go.
 
+Watch a 2 minute video segment on [how to edit a PDF](https://documate.wrkflows.io/en/articles/1007) with [Documate](https://documate.wrkflows.io/)'s [online free PDF editor](https://www.documate.org/pdf).
```

Comment:
I'd leave off links that have been linked in the same section, just so people don't think it's supposed to be new content. Gonna push a change for that in a sec.